ocaml/libs/tracing/tracing.mli

```diff
@@ -129,6 +129,8 @@ module Export : sig
       val set_max_file_size : int -> unit
 
       val set_trace_log_dir : string -> unit
+
+      val set_compress_tracing_files : bool -> unit
```

Comment:
It's not possible right now to observe the status of this flag, only to set it. Maybe you don't need it right now.